core/ir/instr_api.h

```diff
@@ -1995,6 +1995,22 @@ DR_API
 instr_t *
 instr_convert_short_meta_jmp_to_long(void *drcontext, instrlist_t *ilist, instr_t *instr);
 
+DR_API
+/**
+ * Returns the operation size of \p instr if it's a DR_ISA_REGDEPS instruction, OPSZ_NA
```

Comment:
Use # for both DR_* and OPSZ* for links.

Reply:
Done.